sklearn/metrics/tests/test_ranking.py

```diff
@@ -1468,3 +1469,62 @@ def test_partial_roc_auc_score():
         assert_almost_equal(
             roc_auc_score(y_true, y_pred, max_fpr=max_fpr),
             _partial_roc_auc_score(y_true, y_pred, max_fpr))
+
+
+@pytest.mark.parametrize('y_true, k, normalize, sample_weight, true_score', [
```

Comment:
Let's simplify this a bit: 4 classes should be enough, maybe 3?

also maybe group the test by values of k?